docs/cli/tkgctl/bom.md

```diff
@@ -0,0 +1,126 @@
+# Details on BoM usage in the TKG library
+
+## Introduction
+
+BoM stands for Bill of Materials and there are 2 types of BoM files:
```

Comment:
How about putting an explanation at the very start on what BoM files are used for?
This would set the context for the rest of the doc